inception-project/inception
# #977 - Consider alternative labels

**What's in the PR**
- Added option to match against additional properties to in the KB
- Make adjustments in the query builder to consider these additional properties while still returning the main label property if possible
- Added a test which matchers against additional properties

**How to test manually**
* Create a project with a knowledge base
* Set the Named Entity identifier feature to accept "any" type of KB item, not just instances
* Create a new property and call it e.g. "altLabel"
* Create a new class with some label
* Manually create an "altLabel" statement in the class
* Copy the IRI of the altLabel property from the "i" icon tooltip and add it to the additional search properties in the KB settings - make sure to save the settings
* Open a document for annotation
* Create a named entity
* In the identifier field, enter the label you set as altLabel or the one you set as label
* Check that the concept pops up

**Automatic testing**

**Documentation**
